### DISCUSSION TITLE
Request for Examples: Implementing Automatic Dark/Light Theme Switching in NativeWind UI and Tamagui

### DISCUSSION THREAD

**Discussion by uyriq** [danstepanov/create-expo-stack#101] [Q&A] [answered]
It would be great to see examples of an app template with an automatic dark/light theme for both NativeWind UI and Tamagui.

**Comment by danstepanov** [comment, marked as answer]
This will be part of the paid starter we are releasing at the end of the month, it will use NativeWind. That project will be more intended for starting an actual app/company around whereas create expo stack will be for rapid prototyping. Would this address your request?

**Comment by danstepanov** [comment]
There is now dark/light mode switching in NWUI :) No plan to add it for tamagui unless they step in and add it
